### PR TITLE
Implement CRUD for Floating Tasks

### PR DESCRIPTION
**Pull request by nightism**

### CODE REVIEW

**Comment by nightism** [comment]
@pyokagan please help to fix the existing bugs, thanks!

**Comment by pyokagan** [comment]
@nightism you were supposed to do deadline tasks....

**Comment by pyokagan** [comment]
Merged as 2f8b9ba4881c468554f44b679bb81d4b09b7a00d

File: src/main/resources/view/FloatingTaskListCard.fxml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+<VBox xmlns:fx="http://javafx.com/fxml/1" alignment="CENTER_LEFT" maxHeight="120.0" minHeight="80.0"
+        prefHeight="110.0" stylesheets="@DarkTheme.css" styleClass="card">
+    <HBox spacing="5" alignment="CENTER_LEFT">
+        <Label fx:id="indexLabel" styleClass="cell_big_label"></Label>
+        <Label fx:id="nameLabel" styleClass="cell_big_label"></Label>
+    </HBox>
+    <HBox spacing="5" alignment="CENTER_LEFT">
+        <Label>Priority: </Label>
+        <Label fx:id="priority">$priority</Label>
```
Comment (pyokagan) [inline src/main/resources/view/FloatingTaskListCard.fxml:14]: @nightism `<Label fx:id="priorityLabel">$priority</Label>`

**Pull request closed without merging**